test/end-to-end/indexing.test.ts

```diff
@@ -494,6 +465,12 @@ describe('Indexing', function () {
         },
       });
 
+      await waitForLogMessage({
+        container: compose.getContainer('core-1'),
+        expectedMessage: 'Transaction imported.',
+      });
+
+      // wait 10 seconds after transaction was imported to give it time to be indexed  (not be indexed in this case)
```

Comment:
@djwhitt Not exactly sure what to do here to make sure tx was imported and bundle not indexed...

Reply:
Eh, we can leave as is for now. We might tag this one on in the future though.